java/ql/src/experimental/Security/CWE/CWE-489/devMode.ql

```diff
@@ -0,0 +1,16 @@
+import java
```

Comment:
You probably have to add [query metadata](https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md).